pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/KeyValueSchemaImpl.java

```diff
@@ -144,11 +144,6 @@ public String getTopicName() {
         }
     }
 
-    @Override
```

Comment:
Can't this method be deleted?
Because [decode](https://github.com/apache/pulsar/pull/14775/files#diff-4ad4a66a312bd567cdb8ef7959da9bf77535425b17a49f21950c3b3fde105970L149)(line149) is implemented by `KeyValueSchemaImpl` instead of reusing the parent class implementation.

Reply:
@yuruguo Nice catch. Thanks a lot.